internal/testdata/taggedmodels/nontagged.go

```diff
@@ -0,0 +1,5 @@
+package taggedmodels
```

Comment:
```suggestion
package buildtags
```
`buildtags` is more explicit (!= struct tags).

Also, rename `nontagged` => `untagged`.

Reply:
Done